src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -34,14 +35,22 @@ public OrderController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
-        ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
+    public OrderResponseDto completeOrder(@RequestParam Authentication authentication) {
+        ShoppingCart cart = shoppingCartService
+                .getByUser(userService.findByEmail(authentication.getName()).orElseThrow(
+                        () -> new RuntimeException("Can`t add new order for user: "
```

Comment:
message -> Can't find a user by email: authentication.getName()
the same for other similar cases